### DISCUSSION TITLE
Include OIDC compliant JWT issuer like FusionAuth to Supabase Auth

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#28762] [Feature Requests]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

Include [FusionAuth](https://fusionauth.io/) ass one of the third-party authentication providers.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to https://supabase.com/dashboard/project/project123/settings/auth
2. Try to add third-party provider other than Auth0, Firebase, or AWS Cognito.

## Expected behavior

Expect to be able to give any third-party provider. I would expect that the same type of providers list that is accepted in (Auth.js)[https://authjs.dev/reference/core/providers/fusionauth] would be allowed for Supbase as long as the criteria is met for OIDC. There is an open [issue](https://github.com/nextauthjs/next-auth/issues/10867) on this repo as well that I hope gets merged to make the oidc the default listing. 

## Screenshots
![image](https://github.com/user-attachments/assets/68e54bf1-904e-4090-b546-38250c273bf8)

## Additional context

https://twitter.com/codercatdev/status/1824170613700985288

**Comment by encima** [comment]
Hi @alex-fusionauth 
Thanks for opening. As this is a feature request, let me convert this to a discussion and move it under the feature requests category.

TPA is a new feature released during launch week and the initial support is for those providers listed currently, though we welcome new suggestions!
